src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -179,50 +179,48 @@
                DisplayName="5" />
   </EnumProperty>
 
+  <!-- TODO: Create fwlink for this HelpUrl -->
   <StringProperty Name="NoWarn"
-                  DisplayName="Suppress warnings"
-                  Description="Blocks the compiler's ability to generate one or more warnings. Separate multiple warning numbers with a comma (',') or semicolon (';')."
-                  HelpUrl="https://go.microsoft.com/fwlink/?linkid=2147300"
+                  DisplayName="Suppress specific warnings"
+                  Description="Blocks the compiler from generating the specified warnings. Separate multiple warning numbers with a comma (',') or semicolon (';')."
+                  HelpUrl="https://docs.microsoft.com/en-us/visualstudio/ide/how-to-suppress-compiler-warnings"
                   Category="ErrorsAndWarnings" />
 
-  <EnumProperty Name="TreatWarningsAsErrors"
-                HelpUrl="https://go.microsoft.com/fwlink/?linkid=2147301"
```

Comment:
This fwlink is working for me and redirects to the expected compiler documentation for this topic.

In general if there is a problem with a fwlink we should update the forwarding, rather than replacing the fwlink, as this allows us to fix the redirection for shipped versions of the product as well.

Reply:
Interesting. I had tested all of the fwlinks and they did not work. These links are different from the ones I tested. Were they added recently?

Reply:
I'm only aware of the fwlinks I've created for the property pages. Those were created back in Oct. 2020 and haven't been updated since then.

The ones I created are jointly owned by me and @jjmew. Ideally we would set up a security group to own them but I haven't gotten around to that.

Reply:
Huh. I looked back at my Chrome history and the link failures were similar to this: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/errors-warning
Either way, I'll revert these changes.